docs/dev-standards/measuring-security-maturity.md

```diff
@@ -0,0 +1,37 @@
+# Measuring security maturity
+
+## Introduction
+
+Measuring security maturity helps teams understand their current security posture and identify areas for improvement. Regular assessments support continuous improvement and help prioritise investment in training, tooling and secure development practices.
+
+## Guidance
+
+Teams **MUST**:
+
+- complete a security self-assessment every quarter
```

Comment:
This section feels a little light on guidence, given the MUST requirements to self-assess, from what I can tell those frameworks seem to be a set of principals/best practice not a mechanism for self assessment. How will one score themselves against these frameworks? what is the scoring crieria etc? The MVSP mentions self assessing anually where as this suggests quarterly.

I feel that if we want to set out our stall that these are the standards, and expect adoption / adherence then we have to make it easy to understand the requirement and how it could be acheived.

Reply:
it might, it might not be, it depends on the other stuff i.e. scoring, criteria, how to self assess.